scripts/apps/authoring/authoring/controllers/AssociationController.ts

```diff
@@ -210,8 +210,6 @@ export function AssociationController(config, send, api, $q, superdesk,
             return renditions.crop(item, cropOptions)
                 .then((rendition) => {
                     self.updateItemAssociation(scope, rendition, options.customRel, callback);
-                }, () => {
```

Comment:
`renditions.crop` performs an `api.save('picture_crop'...)` on the item. If the api returns an error, should this error be displayed to the user (interpreting the api error response)?

Reply:
yep, will add the notification to the api call